x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/crawler/crawl_requests_panel/crawl_requests_panel.tsx

```diff
@@ -9,16 +9,47 @@ import React from 'react';
 
 import { useValues } from 'kea';
 
-import { EuiCode, EuiPanel, EuiSpacer, EuiText } from '@elastic/eui';
+import { EuiButton, EuiCode, EuiPanel, EuiSpacer, EuiText } from '@elastic/eui';
+import type { DiscoverStart } from '@kbn/discover-plugin/public';
 import { i18n } from '@kbn/i18n';
+import { useKibana } from '@kbn/kibana-react-plugin/public';
 
 import { DataPanel } from '../../../../../shared/data_panel/data_panel';
 import { CrawlerLogic } from '../crawler_logic';
 
 import { CrawlRequestsTable } from './crawl_requests_table';
 
+interface KibanaDeps {
```

Comment:
suggestion(non-blocking):  Might be useful to pull this interface out, move it to `x-pack/plugins/enterprise_search/common/types` and give it a full dependency definition. I expect we'll be using this more in the near-ish future.